src/main/java/haveric/recipeManager/tools/Version.java

```diff
@@ -52,6 +55,19 @@ private static boolean supports1_11() {
 
         return supports;
     }
+    private static boolean supports1_10() {

```

Comment:
Was there something the 1.10 checks were added for? I'm not seeing it being used anywhere.